learning/tour-of-beam/frontend/lib/models/module.dart

```diff
@@ -16,28 +16,29 @@
  * limitations under the License.
  */
 
-import 'package:json_annotation/json_annotation.dart';
 import 'package:playground_components/playground_components.dart';
 
-import 'abstract_node.dart';
+import '../repositories/models/module.dart';
+import 'node.dart';
+import 'parent_node.dart';
 
-part 'module.g.dart';
-
-@JsonSerializable(createToJson: false)
-class ModuleModel {
-  final String id;
-  final String title;
+class ModuleModel extends ParentNodeModel {
```

Comment:
This allows to make the tree widgets agnostic of whether they work with groups or modules, this is useful when handling clicks on any node.